<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>汉字转拼音</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="txtIn"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnToFan">
        <property name="text">
         <string>简转繁</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnToJian">
        <property name="text">
         <string>繁转简</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnToPinYin">
        <property name="text">
         <string>汉字转拼音</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>音调</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbTone">
        <item>
         <property name="text">
          <string>带音调</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>数字音调</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>不带音调</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>分隔符</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbSeparator"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>输出形式</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbLayout">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>不带汉字</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>汉字在下侧</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>汉字在左侧</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>汉字在右侧</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="txtOut"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
